static-assets/components/cstudio-common/common-api.js

```diff
@@ -2535,16 +2535,16 @@ var nodeOpen = false,
        * =>=>=>=> viewcontroller-in-context-edit.initializeContent
        * =>=>=>=>=> constructUrlWebFormSimpleEngine
        */
-      editContent: function (formId, site, id, nodeRef, path, asPopup, callback, auxParams, mode, isFlattenedInclude) {
+      editContent: function (formId, site, mimeType, nodeRef, path, asPopup, callback, auxParams, mode, isFlattenedInclude) {
```

Comment:
Looks dangerous to change this function's signature. Make sure to find in files all possible forms of function use (e.g. `.editContent`, `['editContent']`, `blah.blah.editContent`, etc).

Reply:
done searching for all possible forms of function use, all good